### DISCUSSION TITLE
Implementing Ideal-Image-Slider

### DISCUSSION THREAD

**Discussion by BerkhanEminsoy** [jekyll/jekyll#8893] [Q&A]
Hiya! I'm trying to implement an image slider on my site, more specifically the [Ideal-Image-Slider w/ include](https://github.com/jekylltools/jekyll-ideal-image-slider-include). I've got most of the basics down *except* the **excerpt**... jokes...:grimacing:

Anyhow, what I'm trying to say is that the slider loads fine in the [post page](https://berkhaneminsoy.github.io/minima/projects/2021-11-25-losPinos-jekylliis.html), yet does not load in excerpts on the [home page](https://berkhaneminsoy.github.io/minima/work/).
I get an `unexpected token` error on line 199 where, when my [`slider_scripts.html`](https://github.com/BerkhanEminsoy/minima/blob/43cbaad340d6db7fcbc987f63bf11928d7565536/_includes/slider_scripts.html) include is called, `slider.selector` is not defined or turning up `null`, even though the page `work.md` has `image_sliders_load_all` set to `true`. Both `iis_slider_array` and consequent `selector` prints in the index, yet `console.log(iis_slider_array)` turns up `not defined`.

I'm not sure how to get around this issue at this point, but feels like a quick/dumb fix. Any help is very very much appreciated and could involve supportive compensation! And sorry I can't be more specific or technical! 

[link](https://berkhaneminsoy.github.io/minima/) to live page index

**Comment by pskadasi** [comment]
im facing the same error, have you addressed it, please let me know
